input/examples/condition-noknown.xml

```diff
@@ -4,20 +4,6 @@
     <meta>
         <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-condition"/>
     </meta>
-    <clinicalStatus>
```

Comment:
My error in asking for this one to have clinicalStatus removed. Replace clinicalStatus, otherwise fails due to FHIR constraint.

Reply:
Done.